Organic/Organic.php

```diff
@@ -1,124 +1,24 @@
 <?php
 
-namespace Empire;
+namespace Organic;
 
 use DateTime;
-use Empire\SDK\EmpireSdk;
+use Organic\SDK\OrganicSdk;
 use Exception;
 use WP_Query;
 
 use function \get_user_by;
 use function Sentry\captureException;
 
-const CAMPAIGN_ASSET_META_KEY = 'empire_campaign_asset_guid';
-const GAM_ID_META_KEY = 'empire_gam_id';
-const SYNC_META_KEY = 'empire_sync';
-
-class BaseConfig {
-
-    /**
-     * Map (key -> config-for-placement) of configs for Placements
-     */
-    public array $forPlacement;
-
-    /**
-     * Raw Config returned from Empire Platform API
-     */
-    public array $raw;
-
-    public function __construct( array $raw ) {
-        if ( empty( $raw ) ) {
-            $this->forPlacement = [];
-            $this->raw = [];
-            return;
-        }
-
-        $forPlacement = array_reduce(
-            $raw['placements'],
-            function( $byKey, $config ) {
-                $byKey[ $config['key'] ] = $config;
-                return $byKey;
-            },
-            []
-        );
-
-        $this->forPlacement = $forPlacement;
-        $this->raw = $raw;
-    }
-}
-
-class AdsConfig extends BaseConfig {
-
-    /**
-     * List of AdRules returned from Empire Platform API
-     * Each AdRule must contain at least:
-     *  bool enabled
-     *  string component
-     *  string comparator
-     *  string value
-     */
-    public array $adRules;
-
-    /**
-     * Map (key -> Placement) of Placements returned from Empire Platform API
-     * Each Placement must contain at least:
-     *  array[string] selectors
-     *  int limit
-     *  string relative
-     */
-    public array $forPlacement;
-
-    public function __construct( array $raw ) {
-        parent::__construct( $raw );
-        if ( empty( $raw ) ) {
-            $this->adRules = [];
-            return;
-        }
-
-        $this->adRules = $raw['adRules'];
-    }
-
-    public function getPrebidBuildUrl() : string {
-        $prebid = $this->raw['prebid'] ?? [];
-        return $prebid['useBuild'] ?? 'https://empirecdn.io/assets/prebid5.13.0.js';
-    }
-}
-
-
-class AmpConfig extends BaseConfig {
-
-    /**
-     * Map (key -> amp) of amps for placements returned from Empire Platform API
-     * Each amp must contain at least:
-     *  string html
-     */
-    public array $forPlacement;
-}
-
-
-class PrefillConfig extends BaseConfig {
-
-    /**
-     * Map (key -> prefill) of prefills for placements returned from Empire Platform API
-     * Each prefill must contain at least:
-     *  string html
-     *  string css
-     */
-    public array $forPlacement;
-}
-
-
-class LogLevel {
-    const WARNING   = 'warning';
-    const INFO      = 'info';
-    const DEBUG     = 'debug';
-}
+const CAMPAIGN_ASSET_META_KEY = 'organic_campaign_asset_guid';
+const GAM_ID_META_KEY = 'organic_gam_id';
+const SYNC_META_KEY = 'organic_sync';
```

Comment:
I think these post meta keys should be handled in a compatible manner, similar to `$this->getOption`, and be taken into account in a couple of SQL queries later in the code.

Even better would be to add some one-time script that can be run manually to migrate options and postmetas to the new keys